docs/kafka/manage-cluster/README.adoc

```diff
@@ -0,0 +1,252 @@
+
+.Procedure
+
+. In a terminal, log in to `rhoas`.
++
+--
+[source,shell]
+----
+$ rhoas login
+----
+
+Your web browser opens the {rh-sso-url}[Red Hat Single Sign-On page^].
+--
+
+. Enter your credentials to log in to your Red Hat account.
++
+--
+Welcome pages in the browser notify you that you're logged in to `rhoas` successfully.
+
+In your terminal, the `rhoas login` command indicates that you're logged in.
+--
+. In your terminal, register your ROSA cluster using the `openshift-cluster (oc) register-cluster` command.
++
+--
+[source,shell]
+----
+$ rhoas kafka oc register-cluster
+----
+
+This command installs the following add-ons to the cluster:
+
+* Managed Kafka kas-fleetshard Operator: Required for provisioning and managing instances of Apache Kafka on your {openshift} cluster.
+* Managed Kafka Operator: Required to manage Apache Kafka clusters and related components on your {openshift} cluster.
+* Observability Operator: Required for metrics collection on the Kafka instance. Unlike the previous two, this add-on is displayed on the *Projects* page instead of the *Add-ons* page of the https://console.redhat.com/openshift[OpenShift Cluster Manager Hybrid Cloud Console^].
+--
++
+[NOTE]
+--
+These add-ons are required by and managed by {product-kafka} only and must not be adapted or changed in any way.
+--
++
+. The CLI lists the available {openshift} clusters in your organization  and prompts you to select a ready cluster to register. Use the arrows on your keyboard to select the required cluster.
+. The CLI prompts you to create a machine pool with a specified node count. The required node count is calculated by identifying the number of requested Kafka streaming units and multiplying by three:
++
+----
+required_node_count = requested_Kafka_streaming_units × 3
+----
++
+For example, a Kafka instance of 2 streaming units requires a machine pool of 6 nodes. A streaming unit determines the default maximum capacity of a Kafka instance. An instance with a larger size can handle higher loads and process more events, has more storage, and can handle more clients and connections.
++
+For more information on streaming units and for pricing details, see https://www.redhat.com/en/technologies/cloud-computing/openshift/openshift-streams-for-apache-kafka/pricing[Pricing for Red Hat OpenShift Streams for Apache Kafka^].
++
+Enter `6` as the required node count to create a machine pool with 6 nodes called `kafka-standard`.
++
+[NOTE]
+--
+This kafka-standard machine pool is for use with {product-kafka} only. Do not assign your own production workloads to run within it.
+--
+. Enter `y` to confirm you want your Kafka instances to be accessible through a public network.
+
```

Comment:
How does the reader know what to do next?  What leads them to creating a kafka instance?

Reply:
I can add it in a link to the Getting started guide here.